frontend/src/components/Shared/Map/style/style.json

```diff
@@ -17,7 +17,7 @@
             "collapsed": false
           }
         },
-        "openmaptiles:version": "3.x",
+        "fmaptiles:version": "3.x",
```

Comment:
This line changed after I ran `npm run build-map-style`. Not sure why.

Reply:
That is very odd, build.js in Map/style/scripts doesn't appear to touch that file.